setup.py

```diff
@@ -5,14 +5,15 @@
 
 setuptools.setup(
     name='aussiebb',
-    version='0.2',
+    version='0.3',
     author='Michael Still',
     author_email='mikal@stillhq.com',
     description='A python library to interact with the Aussie Broadband customer portal',
     long_description=long_description,
     long_description_content_type='text/markdown',
     url='https://www.madebymikal.com/aussiebb',
     packages=setuptools.find_packages(),
+    install_requires=['requests'],
```

Comment:
Isn't having requests in requirements.txt sufficient?

Reply:
I had to manually install it.

I'm not a python guru, but I found this one (to help in my own learnings too): https://packaging.python.org/discussions/install-requires-vs-requirements/

I think the most relevant sentence for us in this situation is this one:
```
Whereas install_requires metadata is automatically analyzed by pip during an install, requirements files are not, and only are used when a user specifically installs them using pip install -r.
```

Hope that helps. Thanks so much for this. Gave me a starting point for my project.